### PR TITLE
Fix issues when setting up Emby in the Connect settings

### PR DESCRIPTION
**Pull request by LASER-Yi**
#### Database Migration
NO

#### Description
Fix issues when using Notification with Emby, the documentation can be found in [Emby's document](https://github.com/MediaBrowser/Emby/wiki/User-Authentication#authenticating-a-user)
image

#### Todos
- [ ] Tests
- [ ] Wiki Updates

#### Issues Fixed or Closed by this PR
NO

### CODE REVIEW

File: src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs [hunk 1]
```diff
@@ -22,11 +22,11 @@ public void Notify(MediaBrowserSettings settings, string title, string message)
...
             request.SetContent(new
-                           {
-                               Name = title,
-                               Description = message,
-                               ImageUrl = "https://raw.github.com/NzbDrone/NzbDrone/develop/Logo/64.png"
-                           }.ToJson());
+            {
+                Name = title,
+                Description = message,
+                ImageUrl = "https://raw.github.com/NzbDrone/NzbDrone/develop/Logo/64.png"
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs:28]: Sonarr/Sonarr

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

File: src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs (hunk 1 above)
Comment (LASER-Yi) [reply src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs]: Updated

**Comment by bakerboy448** [comment]
Note: likely breaking API changes for previous legacy versions

**Comment by markus101** [comment]
How long ago did this changes get implemented on Emby's side, does Jellyfin also support this new header and URL Base as well?

**Comment by Qstick** [comment]
Based on what I could dig up, this happened before the JF fork (back in 2018 at some point). Though it was not well documented at all. It seems like JF supports it, but that was my biggest worry in letting this through. I think someone needs to test with JF

**Pull request closed without merging**